sats-dna/src/main/kotlin/com/sats/dna/components/snackbar/SatsSnackbarVisuals.kt

```diff
@@ -0,0 +1,101 @@
+package com.sats.dna.components.snackbar
+
+import androidx.compose.material3.SnackbarDuration
+import androidx.compose.material3.SnackbarVisuals
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.semantics.clearAndSetSemantics
+import com.sats.dna.internal.MaterialIcon
+import com.sats.dna.theme.SatsTheme
+
+class SatsSnackbarVisuals internal constructor(
+    val leadingIcon: @Composable () -> Unit,
+    val title: String?,
+    override val message: String,
+    val action: SatsSnackbarAction?,
+    val dismissAction: SatsSnackbarAction?,
+    override val duration: SnackbarDuration,
+    val backgroundColor: Color,
```

Comment:
Should we call this `containerColor` just because M3 has sort of just went with that name? 🤔